### DISCUSSION TITLE
Camera Raw Filter + Bevel + DPI

### DISCUSSION THREAD

**Discussion by NeticDesigns** [photopea/photopea#5588] [General]
I believe that the Camera Raw Filter should be improved a lot more than it currently is, and not only the Bevel and Emboss feature in Photopea. If you look at Camera Raw Filter compared to Photoshop or other high-quality apps, you'd see a very noticeable difference in design and what you can do with the filter. Also, Bevel and Emboss in Photopea are a bit too rough or taddy, and I believe it's in need of improvement. But it's alright for what it does, also I think the DPI in Photopea should increase or Photopea improving DPI aka resolution,, because sometimes GFX may appear low quality despite you making DPI '500-1000'.

**Comment by photopea** [comment]
Hi, what features exactly would you like to add to Camera Raw? As I work on Photopea alone, I would rather add something people need, than something nobody needs.

Any PSD file with any layer style should look identical when opened in Photoshop or Photopea. Do you have a PSD file which looks differently?

**Reply by NeticDesigns** [reply]
I honestly think that honestly the Clarity Tool should be added in Photopea, I believe it'd heighten the quality of my GFX. Also, the problem isn't with the PSD files, because when my friends open them in both Photoshop and Photopea, they're fine. The problem is when they're exported as PNG files. I saw that when Photoshop exports a PNG file it's way higher quality than the one in Photopea. I also saw this with my friend too who uses Photopea and if you open the PSD, the work is high quality. But when he exports it, the result is a bit low resolution.

**Comment by photopea** [comment]
When you open the same PSD file in Photopea and Photoshop, and you export it as a PNG in both, both PNGs should look identical. 

Could you show us a proof that images from Photopea have a "bad quality" - whatever it means? Exported PNGs have the same resolution as your PSD by default.
